search-index.js

```diff
@@ -0,0 +1,106 @@
+const lunr = require('lunr');
+const redis = require('redis');
+
+const REDIS_PORT = process.env.REDIS_PORT || 6379;
+const REDIS_HOST = process.env.REDIS_HOST || "0.0.0.0";
+// TODO: Add more configuration options for redis
+
+
+// lunrjs indexer
+function lunrIndexer() {
```

Comment:
Since we are using ES6 you might as well use a class.

Reply:
I have changed this.